### DISCUSSION TITLE
Which parts in src directory is the interface to connect to the FPGA?

### DISCUSSION THREAD

**Discussion by allencho1222** [chipsalliance/rocket-chip#3147] [Q&A] [answered]
I hope that it is possible for me to boot Rocket-chip on FPGA except Zynq board(which is currently supported).

Are there any guide to boot Rocket-chip on FPGA that is not Zynq?

If not, which parts should I look at?

**Comment by CLappin** [comment, marked as answer]
Hi,

Yes it is possible to boot the rocket-chip from other FPGA providers.  I'm not sure if there are any guides to doing this. 
The ExampleRocketSystem is the example top level module that you should look for in the generated source code.  

Regards, 
Ciaran
